rclcpp/include/rclcpp/node.hpp

```diff
@@ -972,7 +972,11 @@ class Node : public std::enable_shared_from_this<Node>
 
   /// Return a list of parameters with any of the given prefixes, up to the given depth.
   /**
-   * \todo: properly document and test this method.
+   * \param[in] prefixes The list of prefixes that should be searched for within the
+   * current parameters.
+   * \param[in] depth An unsigned integer that represents the recursive depth to search.
```

Comment:
We should also mention that a depth of ListParameters.RECURSIVE (i.e. 0) will return all parameters.